usr/src/uts/common/fs/zfs/zfs_vnops.c

```diff
@@ -4364,8 +4364,8 @@ zfs_putpage(vnode_t *vp, offset_t off, size_t len, int flags, cred_t *cr,
 }
 
 /*ARGSUSED*/
-void
-zfs_inactive(vnode_t *vp, cred_t *cr, caller_context_t *ct)
+static void
+zfs_inactive_impl(vnode_t *vp, cred_t *cr, caller_context_t *ct)
```

Comment:
are 'cr' and 'ct' used?  If not, I think we can remove those arguments (and /*ARGSUSED*/)